kinova_gen3_7dof/gen3_7dof.xml

```diff
@@ -0,0 +1,121 @@
+              <geom class="visual" mesh="forearm_link" material="white"/>
+              <geom class="collision" mesh="forearm_link_c"/>
+              <body name="spherical_wrist_1_link" gravcomp="1" pos="0 -0.20843 -0.006375" quat="0.707105 -0.707108 0 0">
+                <inertial pos="1e-06 -0.009432 -0.063883" quat="0.703558 0.0707492 -0.0707492 0.703558" mass="0.6781" diaginertia="0.00165901 0.001596 0.000346988"/>
+                <joint class="infiniteJoints" name="joint_5" armature="0.000001928"/>
+                <geom class="visual" mesh="spherical_wrist_1_link" material="white"/>
+                <geom class="collision" mesh="spherical_wrist_1_link_c"/>
+                <body name="spherical_wrist_2_link" gravcomp="1" pos="0 0.00017505 -0.10593" quat="0.707105 0.707108 0 0">
+                  <inertial pos="1e-06 -0.045483 -0.00965" quat="0.44426 0.550121 -0.550121 0.44426" mass="0.6781" diaginertia="0.00170087 0.001641 0.00035013"/>
+                  <joint class="limitedJoints" name="joint_6" range="-2.23 2.23" armature="0.000001928"/>
+                  <geom class="visual" mesh="spherical_wrist_2_link" material="white"/>
+                  <geom class="collision" mesh="spherical_wrist_2_link_c"/>
+                  <body name="bracelet_link" gravcomp="1" pos="0 -0.10593 -0.00017505" quat="0.707105 -0.707108 0 0">
+                    <camera name="wrist_camera" pos="0.0 -0.056 -0.058" fovy="58" mode="fixed" resolution="640 480" quat="0.0 0.0 0.0 1.0"/>
+                    <site name="wrist_camera_optical_frame" pos="0.0 -0.056 -0.058" quat="0.0 0.0 1.0 0.0"/>
+                    <inertial pos="-9.3e-05 0.000132 -0.022905" quat="-0.0368025 0.719402 -0.0444884 0.69219" mass="0.364" diaginertia="0.00024027 0.000222769 0.000213961"/>
+                    <joint class="infiniteJoints" name="joint_7" armature="0.000001928"/>
+                    <geom class="visual" mesh="bracelet_with_vision_link" material="white"/>
+                    <geom class="collision" mesh="bracelet_with_vision_link_c"/>
+                  </body>
+                </body>
+              </body>
+            </body>
+          </body>
+        </body>
+      </body>
+    </body>
+  </worldbody>
+
+  <actuator>
+    <general class="kinova" name="joint_1" joint="joint_1" gainprm="800" biasprm="0 -800 -260"/>
+    <general class="kinova" name="joint_2" joint="joint_2" gainprm="800" biasprm="0 -800 -260"/>
+    <general class="kinova" name="joint_3" joint="joint_3" gainprm="260" biasprm="0 -260 -86"/>
+    <general class="kinova" name="joint_4" joint="joint_4" gainprm="260" biasprm="0 -260 -86"/>
+    <general class="kinova" name="joint_5" joint="joint_5" gainprm="130" biasprm="0 -130 -45"/>
+    <general class="kinova" name="joint_6" joint="joint_6" gainprm="130" biasprm="0 -130 -45"/>
+    <general class="kinova" name="joint_7" joint="joint_7" gainprm="130" biasprm="0 -130 -45"/>
+  </actuator>
```

Comment:
Agreed, I wanted to get the gains off of the kinova we have in the office but their API isn't very user friendly. Also the gains wouldn't be 1-to-1 since there are gears on the actual hardware